src/components/LineChartComponent.tsx

```diff
@@ -1,20 +1,43 @@
 import * as React from 'react';
 import { LineChart } from '@mui/x-charts/LineChart';
 import { useTheme } from '@mui/material/styles';
+import Card from '@mui/material/Card/Card';
+import CardContent from '@mui/material/CardContent/CardContent';
+import Typography from '@mui/material/Typography';
 
-export default function LineChartComponent() {
+export type LineChartData = {
+  id: number
+  value: number
+  label: string
+}
+
+export interface LineChartComponentProps{
+  xAxis: [{}] //e.g. xAxis={[{data: [1, 2, 3, 5, 8, 10]}]}
```

Comment:
Why don't we just put the type of `xAxis` as `number[]` and implement the `LineChart` component as below. Mainly for consistency with `yAxis`

```
<LineChart
            xAxis={[{data: xAxis}]}
            ...
/>
```